DeprecationWarning is raised as an Error

After updating to sleap-io 0.3.0 and running our tests, I ran into a DeprecationWarning that was raised as an error: 
```python
# validate that if original videos, labeled videos, or dimensions are provided, then an equal number of
        # camera devices must be provided.
        # specification of cameras was not allowed in ndx-pose 0.1.*.
        # warn on new, no warning on construction from existing file
        # TODO in a future release, change DeprecationWarning to a ValueError
        original_videos, labeled_videos, dimensions = popargs("original_videos", "labeled_videos", "dimensions", kwargs)
        if original_videos is not None and (devices is None or len(original_videos) != len(devices)):
            if not self._in_construct_mode:
                msg = (
                    "The number of original videos must equal the number of camera devices. This will become an error "
                    "in a future release."
                )
>               raise DeprecationWarning(msg)
E               DeprecationWarning: The number of original videos must equal the number of camera devices. This will become an error in a future release.

/opt/anaconda3/envs/neuroconv_env/lib/python3.12/site-packages/ndx_pose/pose.py:268: DeprecationWarning
```

Instead of raising this case as an error (which causes tests to fail and runs to terminate), could you raise it as a warning instead? i.e.
```python
from warnings import warn
warn(msg, DeprecationWarning)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,6 @@
 # Changelog for ndx-pose
 
-## ndx-pose 0.2.2 (May 7, 2025)
+## ndx-pose 0.2.2 (Upcoming)
 
 ### Bug fixes
 - Treat deprecation warnings as warnings instead of errors. @pauladkisson (#41)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,6 @@
 # Changelog for ndx-pose
 
-## ndx-pose 0.2.2 (Upcoming)
+## ndx-pose 0.2.2 (May 7, 2025)
 
 ### Bug fixes
 - Treat deprecation warnings as warnings instead of errors. @pauladkisson (#41)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,10 @@
 # Changelog for ndx-pose
 
+## ndx-pose 0.2.2 (Upcoming)
+
+### Bug fixes
+- Treat deprecation warnings as warnings instead of errors. @pauladkisson (#41)
+
 ## ndx-pose 0.2.1 (September 26, 2024)
 
 ### Bug fixes

--- a/src/pynwb/ndx_pose/pose.py
+++ b/src/pynwb/ndx_pose/pose.py
@@ -265,21 +265,21 @@ class PoseEstimation(MultiContainerInterface):
                     "The number of original videos must equal the number of camera devices. This will become an error "
                     "in a future release."
                 )
-                raise DeprecationWarning(msg)
+                warnings.warn(msg, DeprecationWarning)
         if labeled_videos is not None and (devices is None or len(labeled_videos) != len(devices)):
             if not self._in_construct_mode:
                 msg = (
                     "The number of labeled videos must equal the number of camera devices. This will become an error "
                     "in a future release."
                 )
-                raise DeprecationWarning(msg)
+                warnings.warn(msg, DeprecationWarning)
         if dimensions is not None and (devices is None or len(dimensions) != len(devices)):
             if not self._in_construct_mode:
                 msg = (
                     "The number of dimensions must equal the number of camera devices. This will become an error in a "
                     "future release."
                 )
-                raise DeprecationWarning(msg)
+                warnings.warn(msg, DeprecationWarning)
 
         pose_estimation_series, description = popargs("pose_estimation_series", "description", kwargs)
         scorer = popargs("scorer", kwargs)

